server/controllers/applicant/index.js

```diff
@@ -1,5 +1,11 @@
+const getApplicantById = require('./getApplicantById');
+const getApplicants = require('./getApplicants');
+const updateApplicant = require('./updateApplicant');
 const getApplicantsStats = require('./getApplicantsStats');
```

Comment:
delete this line from `getApplicantsStats & getApplicants ` from applicant controller cause its already added in admin controller